test/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../.eslintrc",
```

Comment:
Is this `extends` required? I thought it was implicit (but don't know for sure).